routes/v1/posts.js

```diff
@@ -10,26 +10,32 @@ var posts = require.main.require('./src/posts'),
 module.exports = function(middleware) {
 	var app = require('express').Router();
 
-	app.put('/:pid', apiMiddleware.requireUser, function(req, res) {
-		if (!utils.checkRequired(['content'], req, res)) {
-			return false;
-		}
-
-		var payload = {
-			uid: req.user.uid,
-			pid: req.params.pid,
-			content: req.body.content,
-			options: {}
-		};
-
-		if (req.body.handle) { payload.handle = req.body.handle; }
-		if (req.body.title) { payload.title = req.body.title; }
-		if (req.body.topic_thumb) { payload.options.topic_thumb = req.body.topic_thumb; }
-		if (req.body.tags) { payload.options.tags = req.body.tags; }
-
-		posts.edit(payload, function(err) {
-			errorHandler.handle(err, res);
-		});
+	app.route('/:pid')
+		.post(apiMiddleware.requireUser, function(req, res) {
```

Comment:
The HTTP verb seems to have changed here :wink: 

Updating assets should be via `PUT`, new additions via `POST`